odoo/addons/base/tests/test_ir_actions.py

```diff
@@ -383,3 +383,63 @@ def test_unlink_with_dependant(self):
         # uninstall mode: unlink dependant fields
         field.with_context(_force_unlink=True).unlink()
         self.assertFalse(dependant.exists())
+
+    def test_related_field(self):
+        """ create a related field, check filled values"""
+        SOURCE_MODEL = 'res.users'
+        SOURCE_FIELD = 'login'
+
+        TARGET_MODEL = 'res.partner'
+        target_model = self.env['ir.model'].search([('model', '=', TARGET_MODEL)])
```

Comment:
There is a cached method for that
```suggestion
        target_model = self.env['ir.model']._get(TARGET_MODEL)
```